app/javascript/packs/components/sql_form.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import FormRenderer from '@data-driven-forms/react-form-renderer/dist/cjs/form-renderer';
+import componentTypes from '@data-driven-forms/react-form-renderer/dist/cjs/component-types';
+
+import FormTemplate from '@data-driven-forms/pf4-component-mapper/dist/cjs/form-template';
+import componentMapper from '@data-driven-forms/pf4-component-mapper/dist/cjs/component-mapper';
+import useSqlStore from '../store/sql_store';
+
+const SqlForm = () => {
+    const setData = useSqlStore((state) => state.setData);
+    const setLoading = useSqlStore((state) => state.setLoading);
+
+    return <FormRenderer
+            componentMapper={componentMapper}
+            FormTemplate={(props) => <FormTemplate {...props} submitLabel="Perform a query" />}
+            schema={{
+                fields: [{
+                    component: componentTypes.TEXTAREA,
+                    name: 'query',
+                    label: 'Perform a query',
+                    initialValue: 'SELECT * FROM sources'
+                }]
+            }}
+            onSubmit={(values) => {
+            setLoading(true);
+            return Rails.ajax({
+                url: '/app/',
```

Comment:
url /app is the default, on server it will be ENV['PATH_PREFIX']/ENV['APP_NAME'], i.e.
- /app/sources-db-reader
- /app/topological-inventory-db-reader
etc.

(the same approach use Topo/Sources API)

Reply:
Changed so it will take PATH_PREFIX and APP_NAME from `.env` file